park_info.clj

```diff
@@ -209,164 +208,169 @@
-        :parking true
-        :parking_extra "by the Exhibition grounds or in Bubeneč"
-        ;---Additional Information---
-        :dogs true
-        :path_type "asphalt, threshing floor"
-        :administered_by "Prague City"
-        :opening_hours "open year-round"
-        ;---Map---
-        :map_url "https://www.praha.eu/public/9e/a/ad/184253_4_stromovka_web.pdf"     
-    }
-    def Vysehrad{
-        ;---Basic Info---
-        :food true
-        :food_extra "Café Citadela, Café na Hradbách, restuarant Rio´s Vyšehrad (www.riorestaurant.cz), restaurant U dělové koule (tel. +420 224 919 970)"
-        :toilet true
-        :toilet_extra nil
-        :POIs "Natural & Cultural heritage site, suitable for family trips with children, beautiful view of prague, musem, Slavín cemetary for significant persons"
-        ;---Children and Sport---
-        :bike_path true
-        :skating false
-        :sports_field true
-        :sports_field_extra: "tennis courts (tel. +420 606 650 717, +420 606 500 604)"
-        :playground true
-        :playground_extra "by St. Martin chapel, Mr. Trnka playpen, by entrance to Gorlice hall"
-        ;---Directions---
-        :public_transport "
-        stop: Vyšehrad metro line(s): C
-        stop: Podolská Vodárna - trams: #3, #16, #17, #21, #52"
-        :GPS_coords "50°3′46.66″ N, 14°25′22.75″ E"
-        :parking true
-        :parking_extra "underground parking of Prague Congress Centre, or in Na Topolce street"
-        ;---Additional Information---
-        :dogs nil
-        :path_type "paved, threshing floor"
-        :administered_by "Natural Heritage Site - Prague City"
-        :opening_hours "open year-round"
-        ;---Map---
-        :map_url "https://www.praha.eu/public/9e/b2/d0/184274_4_vysehrad_mapa.pdf"     
-    }
-)
+    :GPS_coords "50°05′2.93″ N, 14°23′40.88″ E"
+    :parking true
+    :parking_extra "by Strahov stadium"
+    ;---Additional Information---
+    :dogs nil
+    :path_type "asphalt, concrete, paved, threshing floor"
+    :administered_by "Prague City"
+    :opening_hours "Petřín & Astronomy Station: open year-round
+    Outlook tower: January-March, November - (10:00-18:00), April, September, October - (10:00-19:00), May-August - (10:00-22:00)"
+    ;---Map---
+    :map_url "https://www.praha.eu/public/d/77/d/184279_4_petrin_mapa.pdf"      
+})
+
+(def Riegrovy_sady{
+    ;---Basic Info---
+    :food true
+    :food_extra "Restaurant: Versus (tel. +420 222 717 247)"
+    :toilet true
+    :toilet_extra "in restaurants and portable toilet by playground"
+    :POIs "1920s Outlook tower, 1840 monolith, Captivating view of Prague skyline"
+    ;---Children and Sport---
+    :bike_path true
+    :skating true
+    :sports_field true
+    :sports_field_extra "Sokol Vinohrady"
+    :playground true
+    :playground_extra "yes, by the restaurant"
+    ;---Directions---
+    :public_transport "
+    metro: Jiřího z Poděbrad (Line A)
```

Comment:
Non-standard info display. Should be
stop: Jiřího z Poděbrad - metro line(s): A

Reply:
fixed.